plugins/chat/assets/javascripts/discourse/models/chat-thread.js

```diff
@@ -81,4 +87,13 @@ export default class ChatThread {
   get escapedTitle() {
     return escapeExpression(this.title);
   }
+
+  get titleOrDefault() {
+    return (
+      this.escapedTitle ||
+      `${I18n.t("chat.thread.default_title", {
+        thread_id: this.id,
+      })}`
+    );
+  }
```

Comment:
Couldnt we just have title to be: 

```
args.title || ${I18n.t("chat.thread.default_title", { thread_id: this.id })}
```

Multiplying titlesXxxYyyy makes you always wonder which one you should be using, if we can limit ourselves to title and escapedTitle, that would be great.

Reply:
Oh right yeah makes sense 👍